MvvmCross.iOS.Support/Views/MvxExpandableTableViewSource.cs

```diff
@@ -10,73 +10,91 @@
     using System.Linq;
     using UIKit;
 
-    public abstract class MvxExpandableTableViewSource : MvxTableViewSource
-    {
+    public abstract class MvxExpandableTableViewSource<TItemSource, TItem> : MvxTableViewSource where TItemSource : List<TItem>
```

Comment:
This makes the MvxExpandableTableViewSource generic. Can we have a non generic too?